docs/collecting-data/collecting-from-own-applications/java-tracker/migration-guide-v1/index.md

```diff
@@ -0,0 +1,144 @@
+---
+title: "Migration guide: v1"
+date: "2022-03-24"
+sidebar_position: 69
```

Comment:
Not that important but maybe we could put the migration guides under the same folder like in the mobile trackers?

Reply:
+1